### ISSUE TITLE
retry and retry.delay - Where should it be set ?

### ISSUE THREAD

**Issue by jfgosselin** [Impetus/kundera#455]
```
Doing some tests and it seems that the property "retry" is read from the persistence-unit but
the property "retry.delay" is read from the kundera.client.property file. 

E.g. If move <property name="retry" value="true" /> in kundera_client_properties.xml, it's ignored.

<persistence-unit name="testing_cassandra_storage_pu">
    <provider>com.impetus.kundera.KunderaPersistence</provider>
    <properties>
        <property name="kundera.dialect" value="cassandra" />
        <property name="kundera.keyspace" value="testing" />
        <property name="retry" value="true" />
        <property name="kundera.client.property" value="kundera_client_properties.xml" />
    </properties>
</persistence-unit> 
```

Thanks

JF

**Comment by jfgosselin** [comment]
Not an issue, but any reasons why these params need to be set in different places (retry in persistence-unit and retry.delay in the kundera.client.property file) ?

Thanks

**Comment by mevivs** [comment]
Any such properties  in persistence.xml should be ignored. Client specific property is right place for it.

-Vivek

**Comment by jfgosselin** [comment]
CassandraHostConfiguration

```
protected void buildHosts(String hosts, String portAsString, List<Host> hostsList)
{
.
.


        setConfig(cassandraHost, persistenceUnitMetadata.getProperties(), externalProperties);
.



@Override
protected void setConfig(Host host, Properties props, Map puProperties)
{ 
.
.
.
 hostRetry = Boolean.parseBoolean(props.getProperty(Constants.RETRY) != null ? props.getProperty(
                Constants.RETRY).trim() : null);
```

So it's a bug, if you look at CassandraHostConfiguration, Constants.RETRY is read from the  persistence.xml

It should be setConfig(cassandraHost, externalProperties,  persistenceUnitMetadata.getProperties()); instead ?

**Comment by mevivs** [comment]
Sounds like a Bug. Yes, let me verify and fix this in 2.9

-Vivek

**Comment by mevivs** [comment]
Fixed in current trunk. Please verify.

-Vivek
